packages/components/src/AmountSummaryCard.tsx

```diff
@@ -26,6 +27,7 @@ export const AmountSummaryCard = ({
   callToAction,
   className = "",
   isLoading = false,
+  isSuccess = false,
```

Comment:
This is breaking the number on the card `Available NAM to Stake`, we should add the `isSuccess` there

![Screenshot 2025-03-10 at 18 18 28](https://github.com/user-attachments/assets/7a0ebd88-82d0-4443-bc65-a25338f2b743)